main.cc

```diff
@@ -1442,14 +1442,21 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
                 qp.invoke_on_all(&cql3::query_processor::stop_remote).get();
             });
 
+            sharded<service::topology_state_machine> tsm;
```

Comment:
Should it be sharded at all? It is valid on shard0 only.

Reply:
Good question. But it was sharded before, every storage_service instance had it. If we don't make it sharded, we would have to make it optional and disengaged on non-zero shards. It would be a bit more complicated to propagate to sharded components.

Reply:
Yes. Lets leave it to a follow up.